### ISSUE TITLE
Dual Spec

### ISSUE THREAD

**Issue by Inyu** [Helios-Empire/Helios-Tracker#212]
So i tried dual spec and it doesnt work right atm. i dced upon changing spec and only 1 point was saved. I will test it a bit more but just a heads up for now

**Comment by dikkedeur** [comment]
okay i will explain....

You set your talent points, and action bars, when you are satisfied.. then you do .spec save 1
you will get a notification if its saved or not, or if there is an error ( cooldown or something )

when its saved, you go to trainer and UNLEARN talent points.
now you have again 61 points..
you can now set again talent points that you want to use on spec 2, so you can switch between them.
set your talents and action bars..

then do .spec save 2 ( remember there is a 1 hour cooldown for saving specs.. to avoid massive load on server )

then you are done.
you can now switch specs by..

.spec change (1 or 2 depending which spec you want to use )
YOU WILL GET A DISCONNECT, thats intended since loading actionbars on the fly is not supported in client 2.4.3 so we force a dc on the client to load the new actionbars and talents.

voila done.

i tested it multiple times and didnt find anything strange.. so if you do, please describe how you reproduce it, then we can check if we got same results

**Comment by Inyu** [comment]
yeah i did that but my points are not saved instead i got 1/0/0 spec 1 it should be 33/0/38 and on spec2 i got 0/0/0 i saved after i unleared and respecced.
i did as u descriped but the points arnt saved
update: i just tried it again did .spec save 2 messenge said it is saved to slot 2 then i unlearned my talents specced again after the cd was off i did .spec save 1 then i tried to switch back so spec 2 with .spec change 2 i dced after i logged back in i had 61 talent points not assigned

**Comment by HEGladios** [comment]
What class are you using?

**Comment by Inyu** [comment]
druid but im still testing just takes awhile cause of the cd

**Comment by HEGladios** [comment]
Have you tried clearing your cache?

**Comment by Inyu** [comment]
yes i just did and then tried to swap specs still 61 free talent points

**Comment by HEGladios** [comment]
Hmm try to save your spec again before changing. So first spend all your points, then .spec save 1/2, then .spec change 1/2

**Comment by HEGladios** [comment]
It's an issue with saving your spec to the database. It somehow only saves a couple of talents, but repeats them a bunch of times.

**Comment by HEGladios** [comment]
Found the problem, I will try to resolve this as soon as possible.

**Comment by HEGladios** [comment]
Issue should be resolved now, please try again.

**Comment by Inyu** [comment]
i tried it again and still got 61 free points after change from 1 to 2 but now there is a message "There is no such subcommand"

**Comment by HEGladios** [comment]
Yeah there was a server crash, so I disabled the commands for now.
